generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -175,6 +179,33 @@ class FireSimSmallSystemConfig extends Config(
   new freechips.rocketchip.subsystem.WithInclusiveCache(nWays = 2, capacityKB = 64) ++
   new chipyard.RocketConfig)
 
+//*****************************************************************
+// Bare Rocket configs to test various bridges
+//*****************************************************************
+// Bare Rocket config with only required peripherals
+// Note: need to run linux with initramfs
```

Comment:
This note doesn't seem accurate since you still have FASED in here. This is where you'd need to use the scratchpad mixin.